### PR TITLE
Feature/transformer generator jdk8

### PR DESCRIPTION
**Pull request by polarene**
## Description

jdk8 compatibility for branch `feature/transformer-generator`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,9 @@ cache:
     - $HOME/.m2
 env:
   global:
-    # HCOM OSS Sonatype jira info
```
Comment (fborriello) [inline .travis.yml]: please restore this file as it was

File: bull-common/pom.xml [hunk 2]
```diff
@@ -17,6 +17,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
```
Comment (fborriello) [inline bull-common/pom.xml]: I don't think this is needed

File: .travis.yml (hunk 1 above)
Comment (polarene) [reply .travis.yml]: done

File: bull-common/pom.xml (hunk 2 above)
Comment (polarene) [reply bull-common/pom.xml]: done

**Pull request merged**